source/reference/minio-server/settings/core.rst

```diff
@@ -0,0 +1,331 @@
+.. _minio-server-envvar-core:
+
+=============
+Core Settings
+=============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+This page covers settings that control core behavior of the MinIO process. 
+
+.. include:: /includes/common-mc-admin-config.rst
+   :start-after: start-minio-settings-defined
+   :end-before: end-minio-settings-defined
+
+Common Settings
+---------------
+
+Volumes
+~~~~~~~
+
+.. tab-set::
+
+   .. tab-item:: Environment Variable
+
+      .. envvar:: MINIO_VOLUMES
+
+         The directories or drives the :mc:`minio server` process uses as the storage backend.
+
+         Functionally equivalent to setting :mc-cmd:`minio server DIRECTORIES`.
+         Use this value when configuring MinIO to run using an environment file.
+
+   .. tab-item:: Configuration Setting
+
+      This setting does not have a configuration variable setting.
+      Use the Environment Variable instead.
+
+Environment Variable File Path
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. tab-set::
+
+   .. tab-item:: Environment Variable
+
+      .. envvar:: MINIO_CONFIG_ENV_FILE
+      
+         Specifies the full path to the file the MinIO server process uses for loading environment variables.
+         
+         For ``systemd``-managed files, setting this value to the environment file allows MinIO to reload changes to that file on using :mc-cmd:`mc admin service restart` to restart the deployment.
+
+   .. tab-item:: Configuration Setting
+
+      This setting does not have a configuration variable setting.
+      Use the Environment Variable instead.
+      
+Workers for Expiration
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. tab-set::
+
+   .. tab-item:: Environment Variable
+      :sync: envvar
+
+      .. envvar:: MINIO_ILM_EXPIRY_WORKERS
+
+         Specifies the number of workers to make available to expire objects configured with ILM rules for expiration.
+         When not set, MinIO defaults to using up to half of the available processing cores available.
+
+   .. tab-item:: Configuration Setting
+      :sync: config
+
+      This setting does not have a configuration variable setting.
+      Use the Environment Variable instead.
+
+Domain
+~~~~~~
+
+.. tab-set::
+
+   .. tab-item:: Environment Variable
+
+      .. envvar:: MINIO_DOMAIN
+
+         Set to the Fully Qualified Domain Name (FQDN) MinIO accepts Bucket DNS (Virtual Host)-style requests on.
+
+         For example, setting ``MINIO_DOMAIN=minio.example.net`` directs MinIO to accept an incoming connection request the ``data`` bucket at ``data.minio.example.net``.
```

Comment:
Not sure, but maybe this was intended to be "incoming connection request _to_ the data bucket" ? (From original text.)